doc/source/structures/communication/connection.rst

```diff
@@ -0,0 +1,76 @@
+.. _connection:
+
+Connection
+==========
+
+Connections represent your ability to communicate with other :struct:`processors <kOSProcessor>` or :struct:`vessels <Vessel>`. You can
+use them to find out whether such communication is currently possible and send messages.
+
+Obtaining a connection
+----------------------
+
+There are 2 types of connections. The first type is used to communicate with other :struct:`processors <kOSProcessor>` within the same vessel.
+You can obtain a connection by using `CONNECTION` suffix. Assuming your vessel has a second processor tagged `'second'` it could look like this::
+
+  SET PROCESSOR TO PROCESSOR("second").
+  SET CONNECTION TO PROCESSOR:CONNECTION.
+
+The second type are connections to other vessels. Assuming you have a rover on duna named `'dunarover'` you could obtain a connection to it like this::
+
+  SET VESSEL TO VESSEL("dunarover").
+  SET CONNECTION TO VESSEL:CONNECTION.
```

Comment:
I have a similar comment here as the one above.  Using the same identifier, `vessel`, as both the name of the built-in function and the name of a user's variable doesn't really help distinguish the difference between them in the example.